src/components/charts/LineChartMetric.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropType from 'prop-types';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Legend,
+  Tooltip,
+} from "recharts";
+import { getColorFromGradient } from '../../utils/colors';
+const truncate = (str, n) => (str.length > n) ? str.substr(0, n-1) + '…' : str;
+
+const LineChartMetric = ({ width = 260, height = 140, series, data, showLegend }) => {
+  const mgBottom = showLegend ? 0 : -10;
+  
+  if (data && !Array.isArray(data[0])) {
+    series = [{ name: 'Line', data }];
+  }
+  
+  const lines = series.map((item, idx) => ({
+    key: item.name,
+    dataKey: `value`,
+    data: item.data,
+    name: truncate(item.name, 20),
+    color: getColorFromGradient('rgb(255,233,36)', 'rgb(242,11,11)', (idx + 1) / 5)
+  }));
+
+  if (showLegend) height += (lines.length * 15);
+
+  return (
+    <LineChart width={width} height={height}
```

Comment:
I can only visit the comparison mode by visiting a pre-built comparison link directly. If I do the normal thing of visiting `/`, adding feature to compare, and then clicking "Show Comparison", I get a blank screen and console errors like `TypeError: undefined is not a function (near '...displayedData.map...')`.